builder-function.Dockerfile.j2

```diff
@@ -16,5 +16,6 @@ COPY Makefile .
 
 # and the app itself
 COPY tsconfig.json ./
```

Comment:
Nit: L18-L20 Can be `COPY tsconfig.json src .eslintrc.js ./`?

Reply:
Will check and create a new PR if possible